scripts/update-ts-configs-constants.js

```diff
@@ -61,8 +61,13 @@ function getEsnextTsConfig(pkgRoot, projectRoot) {
   };
 }
 
+function toPosix(p) {
```

Comment:
Maybe a jsdoc comment about why we do this would be nice, not sure if the windows path bug is well known or not

Reply:
Added a comment.

It's not a bug. The generated files work perfectly fine if there are `\` inside as windows doesn't care that much about this. It's just to get the same result on windows and linux as the files are committed.